plugins/oss-audio/CMakeLists.txt

```diff
@@ -16,12 +18,7 @@ target_sources(oss-audio PRIVATE oss-audio.c oss-input.c)
 
 target_link_libraries(oss-audio PRIVATE OBS::libobs OSS::OSS)
 
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/oss-platform.h.in oss-platform.h)
-
-target_include_directories(oss-audio PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-
+configure_file(oss-platform.h.in oss-platform.h)
```

Comment:
Add a newline after?

Reply:
`configure_file` with immediate `target_sources` are usually kept together in the updated files because they logically are connected as well.

Reply:
Hmm. Okay.